tests/test.sh

```diff
@@ -1,15 +1,32 @@
 #!/bin/bash
 
-if [[ debug == "$1" ]]; then
-  INSTRUMENTING=yes
-  shift
-fi
-echo () {
-  [[ "$INSTRUMENTING" ]] && builtin echo $@
-}
+DIRECTION="$2"
```

Comment:
Now I understand how to set the "direction" ... so this need to go into the documentation somewhere